docs/kane-ai-using-parameters.md

```diff
@@ -0,0 +1,117 @@
+---
+id: kane-ai-using-parameters
+title: KaneAI - Using Parameters
+hide_title: false
+sidebar_label: Parameters
+description: Learn how to use parameters in the KaneAI test cases
+keywords:
+  - lambdatest automation
+  - lambdatest kaneai
+  - kaneai scroll elements
+  - kaneai sidebar scroll
+url: https://www.lambdatest.com/support/docs/kane-ai-using-parameters/
+site_name: LambdaTest
+slug: kane-ai-using-parameters/
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+import {YOUR_LAMBDATEST_USERNAME, YOUR_LAMBDATEST_ACCESS_KEY} from "@site/src/component/keys";
+
+<script type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify({
+       "@context": "https://schema.org",
+        "@type": "BreadcrumbList",
+        "itemListElement": [{
+          "@type": "ListItem",
+          "position": 1,
+          "name": "Home",
+          "item": "https://www.lambdatest.com"
+        },{
+          "@type": "ListItem",
+          "position": 2,
+          "name": "Support",
+          "item": "https://www.lambdatest.com/support/docs/"
+        },{
+          "@type": "ListItem",
+          "position": 3,
+          "name": "KaneAI Parameters",
+          "item": "https://www.lambdatest.com/support/docs/kane-ai-using-parameters/"
+        }]
+      })
+    }}
+></script>
+This guide provides a detailed walkthrough for utilizing parameters in KaneAI to streamline test case management and execute tests with various data inputs. By leveraging data parameters, testers and developers can manage configurations effortlessly. Follow the step-by-step instructions below for successful implementation.
+
+## Key Features of Parameters in KaneAI
+- **Dynamic Data Input :** Use parameters to define and input test data dynamically during test execution.
+- **Multiple Configurations :** Simplify the execution of tests with different data sets.
+- **Seamless Test Management :** Centralized management of parameters improves efficiency and reduces redundancy.
+
+## Step 1: Define Parameters in a Web or App Agent Test
+- Navigate to the specific web agent test URL.
+- Prepare the test to accept inputs for `name`, `age`, and `phone_number` as parameters.
```

Comment:
name, age phone number are just examples. Please treat them as such @amanchopra1905 . Repeating this again.

Update this phrase.